syntax = 'proto3';

message B64Image {
    string b64image = 1;
    int32 width = 2;
    int32 height = 3;
}

enum ImageCategory {
    NOT_CAT = 0;
    CAT = 1;
}

message ClassificationResponse {
    ImageCategory category = 1;
}

service Classification {
    rpc Classify(B64Image) returns (ClassificationResponse);
}
